### PR TITLE
Update code to reflect the el8 and el9 rpms

### LINKED ISSUE #23: Update page to be less out of date
As reported by email:

> the "latest" news dated 2017 suggests your project is dead:
> 
> https://apel.github.io/
> 
> A short blurb about HS23 could be all that is needed there...
> 
> And finally remove references to SL6?
> 

- [x] Add HS23 blurb
- [x] Remove SL6 bits

### PR DESCRIPTION
**Pull request by Sae126V**
Resolves #23 

This PR is about reflecting the el8 and el9 rpms.

### CODE REVIEW

**Comment by Sae126V** [comment]
regarding https://github.com/apel/apel.github.io/pull/27/commits/d99740f86fbcad65c092199a5e8d23ffc163b4b0 ;

It seems rvm `2.7.4` uses gem version `3.1.6` which is causing an issue to install `ffi`- Ruby FFI library.

Upgrading to rvm `3.3.4` it uses gem version `3.5.11`. It should install `ffi` without any issues.

**Comment by Sae126V** [comment]
Pinned ruby version to  `3.3.3` version because the current stable version 3.3.4 isn't available in 'travis-ci-language-archive' yet.

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 language: ruby
-rvm: 2.7.4
+rvm: 3.3.3 # Pinned it to this version because the current stable
+           # version 3.3.4 isn't available in 'travis-ci-language-archive' yet
```
Comment (tofu-rocketry) [inline .travis.yml:3]: Can you use either a block comment before the relevant code line, or a single in-line comment? No hybrids like this, please.

**Review by tofu-rocketry — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (Sae126V) [reply .travis.yml]: Sure Adrian, Removed this in https://github.com/apel/apel.github.io/pull/27/commits/e42af99e0486edc2bfe8381ca5daa56b604637f6.

**Review by tofu-rocketry — APPROVED** [review]

**Pull request merged**